packages/ui-react/src/hooks/useFilteredContent.ts

```diff
@@ -0,0 +1,92 @@
+import type { Content } from '@jwp/ott-common/types/config';
+import useBreakpoint, { Breakpoint } from '@jwp/ott-ui-react/src/hooks/useBreakpoint';
+
+const DEVICE_FILTER_LABELS = {
+  mobile: 'mobile-device',
+  tablet: 'tablet-device',
+  desktop: 'desktop-device',
+};
+
+const COUNTRIES = {
+  DE: 'Germany',
+  FR: 'France',
+  NL: 'Netherlands',
+  RO: 'Romania',
+};
+
+const TIMEZONES = {
+  'Europe/Berlin': {
+    u: 60,
+    d: 120,
+    c: ['DE'],
+  },
+  'Europe/Paris': {
+    u: 60,
+    d: 120,
+    c: ['FR'],
+  },
+  'Europe/Amsterdam': {
+    u: 60,
+    d: 120,
+    c: ['NL'],
+  },
+  'Europe/Bucharest': {
+    u: 120,
+    d: 180,
+    c: ['RO'],
+  },
+};
+
+const useDeviceType = () => {
+  const breakpoint = useBreakpoint();
```

Comment:
I think there is no need for use to use this hook here. Take a look at how it is implemented, it uses `getScreenSize` function under the hood. I suggest we only take this function as we don't really need event handlers.

Reply:
Nice catch, I updated the code 🚀